build.sh

```diff
@@ -148,4 +148,12 @@ echo Generating $BUILD_TYPE makefiles
 cmake $CMAKE_GEN $CC_PREFIX $ICU_PATH -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
 
 $MAKE $MULTICORE_BUILD 2>&1 | tee build.log
+
+result=$?
+if [[ $result == 0 && ! -f ../ch ]]; then
+    ln -s BuildLinux/ch ../ch
```

Comment:
I agree with the above concerns. It's fairly straightforward to know where the test binary is. Especially as long as this is intended to use with already-written test code, saving typing is a moot point. For manual testing, there's not that much extra typing to do, and a wrapper script might be more useful.